vaxrank/cli.py

```diff
@@ -278,6 +281,10 @@ def main(args_list=None):
         arg_parser = new_run_arg_parser()
 
     args = arg_parser.parse_args(args_list)
+    logging.config.fileConfig(
+        pkg_resources.resource_filename(__name__, 'logging.conf'),
+            defaults={'logfilename': args.log_path})
```

Comment:
Glad developers thought about this use case and exposed the API. Hope it wasn't one of those hidden features that caused you trouble to get it working.